charts/internal/shoot-system-components/charts/csi-driver-manila/templates/daemonset-node.yaml

```diff
@@ -0,0 +1,229 @@
+              port: healthz
+            initialDelaySeconds: 30
+            timeoutSeconds: 10
+            periodSeconds: 30
+{{- if $.Values.resources.driverNode }}
+          resources:
+{{ toYaml $.Values.resources.driverNode | indent 12 }}
+{{- end }}
+          volumeMounts:
+            - name: {{ .protocolSelector | lower }}-fwd-plugin-dir
+              mountPath: /csi
+            - name: pods-mount-dir
+              mountPath: /var/lib/kubelet/pods
+              mountPropagation: "Bidirectional"
+
+        - name: {{ .protocolSelector | lower }}-registrar
+          image: {{ index $.Values.images "csi-node-driver-registrar" }}
+          args:
+            - --csi-address=/csi/csi.sock
+            - --kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)
+            - -v=5
+          env:
+            - name: DRIVER_REG_SOCK_PATH
+              value: /var/lib/kubelet/plugins/{{ .protocolSelector | lower }}.manila.csi.openstack.org/csi.sock
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+{{- if $.Values.resources.nodeDriverRegistrar }}
+          resources:
+{{ toYaml $.Values.resources.nodeDriverRegistrar | indent 12 }}
+{{- end }}
+          volumeMounts:
+            - name: {{ .protocolSelector | lower }}-plugin-dir
+              mountPath: /csi
+            - name: registration-dir
+              mountPath: /registration
+
+        - name: {{ .protocolSelector | lower }}-manila-liveness-probe
+          image: {{ index $.Values.images "csi-liveness-probe" }}
+          args:
+            - --csi-address=/csi/csi.sock
+            - --health-port={{ .node.livenessProbe.healthPort }}
+{{- if $.Values.resources.livenessProbe }}
+          resources:
+{{ toYaml $.Values.resources.livenessProbe | indent 12 }}
+{{- end }}
+          volumeMounts:
+            - name: {{ .protocolSelector | lower }}-plugin-dir
+              mountPath: /csi
+
+        - name: {{ .protocolSelector | lower }}-nfs-liveness-probe
+          image: {{ index $.Values.images "csi-liveness-probe" }}
+          args:
+            - --csi-address=/csi/csi.sock
+            - --health-port={{ .node.livenessProbe.healthPort2 }}
+{{- if $.Values.resources.livenessProbe }}
+          resources:
+{{ toYaml $.Values.resources.livenessProbe | indent 12 }}
+{{- end }}
+          volumeMounts:
+            - name: {{ .protocolSelector | lower }}-plugin-dir
+              mountPath: /csi
+
+        {{- end }}
+
+      volumes:
+        - name: registration-dir
+          hostPath:
+            path: /var/lib/kubelet/plugins_registry
+            type: Directory
+        {{- range .Values.shareProtocols }}
+        - name: {{ .protocolSelector | lower }}-plugin-dir
+          hostPath:
+            path: /var/lib/kubelet/plugins/{{ .protocolSelector | lower }}.manila.csi.openstack.org
+            type: DirectoryOrCreate
+        - name: {{ .protocolSelector | lower }}-fwd-plugin-dir
```

Comment:
The protocol plugin-dir does not need to be exposed on the hostpath -> we can `emptyDir` it. There is no registered NFS CSIDriver with the cluster, so there is no need for the kubelet to directly contact this endpoint.